<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>165</width>
      <height>95</height>
     </rect>
    </property>
    <property name="title">
     <string>Volt meter</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>GPIB Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gpibAddVolt">
         <property name="value">
          <number>23</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="valueVolt">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>165</width>
      <height>95</height>
     </rect>
    </property>
    <property name="title">
     <string>Ampere meter</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>GPIB Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gpibAddAmp"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="valueAmp">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>236</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leIp">
       <property name="text">
        <string>192.168.178.153</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
